lib/common/acl.c

```diff
@@ -85,49 +83,37 @@ create_acl(xmlNode *xml, GList *acls, enum xml_private_flags mode)
                   crm_element_name(xml), acl->xpath);
 
     } else {
-        int offset = 0;
-        char buffer[MAX_XPATH_LEN];
+        GString *xpath = g_string_sized_new(64);
```

Comment:
Currently allocating different amounts in different places but it's fairly arbitrary. We could define a constant in a header somewhere like 128 or 256 and just go with that.

Reply:
I think each situation is different, no need to shoehorn into one model